Automattic/wp-calypso
# Plans: add support for jetpack plans, display monthly pricing

This PR adds support for Personal and Jetpack plans to the PlansFeature component.

### Notes
- Jetpack has both monthly plans and annual plans
- We may want to tinker with which features we display for Jetpack and Personal plans
- I've removed a number of warnings on the plan constants page
- We still have outstanding display issues with which buttons should be enabled if your sites already has a plan. (Please ignore this issue for now)
- Upgrade actions are being tackled in #6290
- Jetpack FAQ copy is not ready atm

## Testing Instructions
- Start Calypso with `ENABLE_FEATURES=manage/plan-features make run`
- Navigate to http://calypso.localhost:3000/devdocs/app-components/plan-features
- Verify that PlanFeatures for Personal and Jetpack plans looks correct
- Navigate to http://calypso.localhost:3000/plans
- Select a site
- Verify that PlanFeatures for Personal and Jetpack plans looks correct

Personal:
<img width="529" alt="screen shot 2016-06-22 at 5 21 41 pm" src="https://cloud.githubusercontent.com/assets/1270189/16288213/739d7ff6-389e-11e6-8956-5a9daccad0a1.png">

Jetpack Premium:
<img width="547" alt="screen shot 2016-06-24 at 5 03 36 pm" src="https://cloud.githubusercontent.com/assets/1270189/16353381/44c2b14c-3a2e-11e6-9d7b-e3b52b0bbb59.png">

Jetpack Professional:
<img width="548" alt="screen shot 2016-06-24 at 5 12 05 pm" src="https://cloud.githubusercontent.com/assets/1270189/16353425/df1bbc7a-3a2e-11e6-9104-272d4e2a32bd.png">

Old Jetpack Plans Page For Reference:
<img width="752" alt="screen shot 2016-06-22 at 5 28 27 pm" src="https://cloud.githubusercontent.com/assets/1270189/16288253/c2b77902-389e-11e6-81ef-32d883a20a85.png">

Old Jetpack Compare Page For Reference:
<img width="715" alt="screen shot 2016-06-22 at 5 01 54 pm" src="https://cloud.githubusercontent.com/assets/1270189/16288208/647dc7ec-389e-11e6-9aa5-d62fb7d256c6.png">

cc @lamosty @artpi @rralian @rodrigoi
